### PR TITLE
Template update for nf-core/tools version 1.8

### PR DESCRIPTION
**Pull request by Zethson**
Many thanks to contributing to nf-core/mhcquant!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/mhcquant branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/mhcquant)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by Zethson** [comment]
What a pain...

I will review that again in the coming days, since it is possible, that I accidentally deleted something or stuff from the template that we don't want slipped in.

Please don't merge yet.

**Comment by Zethson** [comment]
`OSError: [Errno 13] Permission denied: '/.local'`

Sudo is however enabled. Not quite sure why there's a permission error.

**Comment by Leon-Bichmann** [comment]
thx  - would be nice to update the TEMPLATE again

File: main.nf [hunk 1]
```diff
@@ -9,21 +9,22 @@
...
+
     Usage:
 
     The typical command for running the pipeline is as follows:
 
     nextflow run nf-core/mhcquant --mzmls '*.mzML' --fasta '*.fasta' --vcf '*.vcf' --class_1_alleles 'alleles.tsv' --include_proteins_from_vcf --predict_class_1 --refine_fdr_on_predicted_subset -profile standard,docker
 
     Mandatory arguments:
-      --mzmls                           Path to input data (must be surrounded with quotes)
+      --mzmls                           Path to input data (must be surrounded with quotes)
```
Comment (Zethson) [inline main.nf]: Time to annotate all other parameters. This will be done in another PR somewhen I guess.

**Comment by Zethson** [comment]
There's a list of other stuff that I want to change, but for now just the template.

**Comment by Zethson** [comment]
I just noticed that 1.9 was pushed here. There apparently are a lot of changes now that we don't want!
This is really annoying.


@ewels I saw that you were wondering about the 404s. MHCquant was one of the repos that had 404s. Maybe they are related to already open PRs or something?

**Comment by Zethson** [comment]
Yeah - it pretty much force merged itself :( Yet again deleted all scripts in /bin and so on.

Maybe it's easiest to go one commit back (no 1.9 update) and trigger that one manually again?

**Comment by Leon-Bichmann** [comment]
I'm lost with all the Template updates and BSL2.

**Comment by Leon-Bichmann** [comment]
Would it make sense to wait with the new release and Template update once everything is clear what will have to be updated after the next nfcore hackathon?

**Comment by Zethson** [comment]
I don't know whether 1.10 will be released after the hackathon.
I would suggest to apply the updates incrementally.

Revert the template commit here to 1.8, apply 1.8.
Next, trigger 1.9 and apply. Then we wait for 1.10.
